xmppserver/src/main/java/org/jivesoftware/openfire/session/LocalIncomingServerSession.java

```diff
@@ -125,6 +125,20 @@ public static LocalIncomingServerSession createSession(String serverName, XmlPul
             toDomain = serverName;
         }
 
+        boolean hasCertificates = false;
+        try {
+            hasCertificates = !connection.getConfiguration().getIdentityStore().getAllCertificates().isEmpty();
+        }
+        catch (Exception e) {
+            Log.error("Unable to load find any content in the identity store. This connection won't be able to support TLS.", e);
```

Comment:
nit: `load` or `find` but not both (I see that was worded this way prior to your change)